test_app/tests/rest_filters/rest_framework/test_field_lookup_backend.py

```diff
@@ -173,3 +175,28 @@ def test_filter_jsonfield_as_text(admin_api_client):
     response = admin_api_client.get(url + '?' + urlencode(query_params))
     assert response.status_code == 200
     assert response.data['count'] == 1
+
+
+def test_filter_unexpected_field(admin_api_client):
+    url = reverse('organization-list')
+    response = admin_api_client.get(url, data={'foofield': 'bar'})
+    assert response.status_code == 400, response.data
+
+
+def test_app_ignore_field(admin_api_client):
+    base_list = tuple(settings.ANSIBLE_BASE_REST_FILTERS_RESERVED_NAMES)
+    with override_settings(ANSIBLE_BASE_REST_FILTERS_RESERVED_NAMES=base_list + ('foofield',)):
+        url = reverse('organization-list')
+        response = admin_api_client.get(url, data={'foofield': 'bar'})
+        assert response.status_code == 200, response.data
```

Comment:
Do you want to parameterize this function to ensure that you can use `foofield`, `cud` and also something from `ANSIBLE_BASE_REST_FILTERS_RESERVED_NAMES`? To ensure that they are properly unioned?